src/components/board/Stack.vue

```diff
@@ -78,7 +82,11 @@
 			@should-accept-drop="canEdit"
 			@drop="($event) => onDropCard(stack.id, $event)">
 			<Draggable v-for="card in cardsByStack" :key="card.id">
-				<CardItem v-if="card" :id="card.id" />
+				<transition :appear="animate && !card.animated && (card.animated=true)"
```

Comment:
you can do assignments inline? I remember looking at it and couldn't see any proper way to do so. Are vuejs guidelines ok with this? :thinking:

Reply:
Yeah, not the nicest but i've seen this in quite some places. Also this part of code is not really the nicest, but I cannot tell the transition to only run when a card if freshly added and not on sorting/initial load. :woman_shrugging:

Reply:
Then I'm closing my eyes :see_no_evil: :stuck_out_tongue_closed_eyes: